src/include/btmem.h

```diff
@@ -387,14 +387,11 @@ struct __wt_page_modify {
 	__wt_spin_unlock((s), &S2C(s)->page_lock[(p)->modify->page_lock])
 	uint8_t page_lock;		/* Page's spinlock */
 
-#define	WT_PM_REC_EMPTY		0x01	/* Reconciliation: no replacement */
-#define	WT_PM_REC_MULTIBLOCK	0x02	/* Reconciliation: multiple blocks */
-#define	WT_PM_REC_REPLACE	0x04	/* Reconciliation: single block */
-#define	WT_PM_REC_REWRITE	0x08	/* Reconciliation: rewrite in place */
-#define	WT_PM_REC_MASK							\
-	(WT_PM_REC_EMPTY | WT_PM_REC_MULTIBLOCK |			\
-	 WT_PM_REC_REPLACE | WT_PM_REC_REWRITE)
-	uint8_t flags;			/* Page flags */
+#define	WT_PM_REC_EMPTY		1	/* Reconciliation: no replacement */
+#define	WT_PM_REC_MULTIBLOCK	2	/* Reconciliation: multiple blocks */
+#define	WT_PM_REC_REPLACE	3	/* Reconciliation: single block */
+#define	WT_PM_REC_REWRITE	4	/* Reconciliation: rewrite in place */
+	uint8_t recon_result;		/* Reconciliation state */
```

Comment:
Sure, renamed and pushed.

I left the field as a uint8_t instead of an enum because an enum is 4B, and I hate to grow WT_PAGE_MODIFY.